### DISCUSSION TITLE
Substitute for `resolveConfig` in 4.0?

### DISCUSSION THREAD

**Discussion by piscopancer** [tailwindlabs/tailwindcss#15496] [General]
![image](https://github.com/user-attachments/assets/199128b4-61da-416f-acb0-4a23b2c85998)

I like intellisense that comes with `resolveConfig` and that I can add new properties and override the type with TS. Since removal in v4, what should I use to access variables in TS?

**Comment by wmodden** [comment]
it was [removed](https://tailwindcss.com/docs/upgrade-guide#theme-values-in-javascript), what could we use now on v4?
